cli/container.go

```diff
@@ -93,12 +99,26 @@ func (c *container) config() (*types.ContainerCreateConfig, error) {
 			IpcMode:       c.ipcMode,
 			PidMode:       c.pidMode,
 			UTSMode:       c.utsMode,
+			Sysctls:       sysctls,
 		},
 	}
 
 	return config, nil
 }
 
+func parseSysctls(sysctls []string) (map[string]string, error) {
+	results := make(map[string]string)
+	for _, sysctl := range sysctls {
+		fields := strings.SplitN(sysctl, "=", 2)
+		if len(fields) != 2 {
+			return nil, fmt.Errorf("invalid sysctl: %s", sysctl)
```

Comment:
Could you provide more invalid reasons? @YaoZengzeng 
I think here we tell user the invalidness of input, but we have not told them why it is invalid which will increase his confusion and time in finding ways to solve this. Maybe more time for us to support these cases. But if we add the invalid reasons. It may reduce much potential work no matter in developing or supporting. How about the format ` fmt.Errorf("invalid sysctl %s: sysctl must be in format of key=value", sysctl)`.

I have to say many places have the same issue. While we need to improve that places. In addition, I think we need to update the cli flag illustration as well, since they are not so readable for users. We code them just in developer's perspectives.

Reply:
@allencloud  can't agree more.